themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -58,27 +58,17 @@ VuFind.register('itemStatuses', function ItemStatuses() {
       el.querySelectorAll('.callnumber,.hideIfDetailed,.location').forEach((e) => e.classList.add('hidden'));
       var locationListHTML = "";
       for (var x = 0; x < result.locationList.length; x++) {
+        var status = result.locationList[x].status;
```

Comment:
I believe this is causing the error. `.status` is undefined. You probably want to check for `.availability`.